src/aleph/vm/hypervisors/firecracker/microvm.py

```diff
@@ -472,12 +472,16 @@ async def teardown(self):
         if self.stderr_task:
             self.stderr_task.cancel()
 
+        # Clean mounted block devices
         if self.mounted_rootfs:
             logger.debug("Waiting for one second for the VM to shutdown")
             await asyncio.sleep(1)
-            root_fs = self.mounted_rootfs.name
-            system(f"dmsetup remove {root_fs}")
-            system(f"dmsetup remove {root_fs}_base")
+            if self.mounted_rootfs.is_block_device():
```

Comment:
Can you add a comment in the code explaining this ?

Reply:
I have put the comment up: https://github.com/aleph-im/aleph-vm/pull/659/files/17d5ab5c511430f0467dceda1a9c12b82fddcd2c#diff-81d0c3cbfe4e4c759d3074c29ddc23188851005f7372b1c8ac83eb1d35f80857R475